user/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+import json, bcrypt, jwt
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+
+from .models      import User
+from my_settings  import SECRET_KEY, ALGORITHM
+
+class SigninView(View):
+    def POST(self,request):
+        try:
+            data     = json.loads(request.body)
+            identity = data ['identity']
+            password = data ['password']
+            
+            if not User.objects.filter(identity = identity).exists():
+                return JsonResponse({'message': 'INVALID_USER'}, status=401) 
+        
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('uft-8')):
+                return JsonResponse({'message': 'INVALID_USER'}, status=401)
+        
+            return JsonResponse({'message': 'SUCCESS', 'user_token' : jwt.encode({'id:user.id'}, SECRET_KEY, ALGORITHM)}, status=200)
```

Comment:
수정 사항이 몇가지 보이네요~

1. 보내는 response에서 `message: SUCESS`는 필요 없을 것 같아요. `access_token`을 받는 것 만으로 이미 성공이니까요~
2. `user_token` 보다는 범용적으로 사용하는 `acess_token`이라는 명칭을 사용하는게 좋을 것 같아요!
3. database에서 user 객체를 꺼내오는 코드가 없는 것 같아요!

```suggestion
    payload      = {"user_id : ..}
    access_token = jwt.encode(payload, SECRET_KEY, ALGORITHM)
    
    return JsonResponse({'access_token' : acess_token}, status=200)
```